src/lib/UtapiClient.js

```diff
@@ -478,6 +478,72 @@ export default class UtapiClient {
     }
 
     /**
+    * Updates counter for the AbortMultipartUpload action
+    * @param {object} params - params for the metrics
+    * @param {string} [params.bucket] - (optional) bucket name
+    * @param {string} [params.accountId] - (optional) account ID
+    * @param {number} timestamp - normalized timestamp of current time
+    * @param {string} action - action metric to update
+    * @param {object} log - Werelogs request logger
+    * @param {callback} callback - callback to call
+    * @return {undefined}
+    */
+    _pushMetricAbortMultipartUpload(params, timestamp, action, log, callback) {
+        this._checkProperties(params, ['byteLength']);
+        const { byteLength } = params;
+        this._logMetric(params, '_pushMetricAbortMultipartUpload', timestamp,
+            log);
+        const paramsArr = this._getParamsArr(params);
+        const cmds = [];
+        paramsArr.forEach(p => cmds.push(
+            ['decrby', generateCounter(p, 'storageUtilizedCounter'),
+                byteLength],
+            ['incr', generateKey(p, action, timestamp)]
+        ));
+        const cmdsLen = cmds.length;
+        const paramsArrLen = paramsArr.length;
+        return this.ds.batch(cmds, (err, results) => {
+            if (err) {
+                log.error('error incrementing counter', {
+                    method: 'UtapiClient._pushMetricAbortMultipartUpload',
+                    error: err,
+                });
+                return callback(errors.InternalError);
```

Comment:
why not returning false like below?  why are you returning booleans from this function? do you use them?  the jsdoc says return is undefined and it is not consistent what is being returned.